docker-files-for-Gitlab-CI-rust/parity-eth-linux-ci/README.md

```diff
@@ -0,0 +1,9 @@
+Docker image based on [RUST](https://www.rust-lang.org/) rust:stretch image (latest stable), [node.js](https://nodejs.org), for [GitLab CI runner](https://gitlab.com/gitlab-org/gitlab-ci-multi-runner).
+Usage:
+```
+linux-stable:
+    stage: build
+    image: parity/rust-linux:stretch
```

Comment:
just as a fincal comment (and too late):
as the dockerfile is tailored for parity-ethereum, probably the docker tag or image string and the documentation should reflect that @TriplEight

Reply:
you mean to change the maintainer, makefile maintainer, that would help someone to fix if something.
But placing a nickname in the container name, it would be at least not informative. I agree that came up with not the best name for the container, maybe you have a better one?

Reply:
well I mean if the containers purpose is entirely to build parity-ethereum, why not call the image rust-parity-ethereum or tag the rust-linux image stretch-parity-ethereum